<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>680</width>
      <height>430</height>
     </rect>
    </property>
<!--    <property name="text">-->
<!--     <string>소요시간:12시간정도</string>-->
<!--    </property>-->
   </widget>

   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>건물구조 보기</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>한 층의 평면도 보기</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>이미 러닝된 데이터 불러오기</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>딥러닝 새롭게 시작</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>딥러닝 모델 보기</string>
    </property>
   </widget>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
